doc/_static/custom_styles.css

```
@@ -136,14 +136,14 @@ div#driver-popover-item .driver-navigation-btns {
 div#driver-popover-item .tour-arrow {
     font-size: 2em;
     line-height: 0;
-    padding-bottom: 0.05em;
+    font-weight: bold;
     pointer-events: none;
     vertical-align: middle;
 }
 
 /* Style tour progress indicator */
 div#driver-popover-item .tour-progress-indicator {
     float: right;
-    font-style: italic;
-    font-weight: normal;
+    font-weight: bold;
+    font-size: 0.6em;
```

This seems pretty small and hard to read to my eye. Maybe a bit bigger?

```suggestion
    font-size: 0.7em;
```